packages/@vuepress/core/lib/client/style/config.styl

```diff
@@ -4,6 +4,9 @@ $textColor = #2c3e50
 $borderColor = #eaecef
 $codeBgColor = #282c34
 $arrowBgColor = #ccc
+$badgeTip = #42b983
+$badgeWarning = darken(#ffe564, 35%)
+$badgeError = #DA5961
```

Comment:
To be consistent with `$accentColor`, `$textColor`, etc. , adding "color" to variable name could be better.

For example: `$badgeTipColor`

Reply:
+1